ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Dragonwell/tasks/main.yml

```
@@ -16,7 +16,11 @@
 
 - name: Download Dragonwell
   win_get_url:
-    url: 'https://github.com/alibaba/dragonwell8/releases/download/ragonwell-8.5.4_jdk8u272-ga/Alibaba_Dragonwell_8.5.4-GA_Experimental_Windows_x64.zip'
+    url: https://github.com/alibaba/dragonwell8/releases/download/dragonwell-8.5.4_jdk8u272-ga/Alibaba_Dragonwell_8.5.4-GA_Experimental_Windows_x64.zip
+    checksum: 0f04bee12ce1acd70d603759e1c57d8381c08669bdeed743474ce9ecfae51fb3
+    checksum_algorithm: sha256
+    checksum: 0f04bee12ce1acd70d603759e1c57d8381c08669bdeed743474ce9ecfae51fb3
+    checksum_algorithm: sha256
```

Looks @sxa 's commit implemented the checksum twice ? 

```suggestion
    checksum: 0f04bee12ce1acd70d603759e1c57d8381c08669bdeed743474ce9ecfae51fb3
    checksum_algorithm: sha256
```